content/en/security/cspm/_index.md

```diff
@@ -11,18 +11,46 @@ Cloud Security Posture Management is not currently available in this site.
+## Detect misconfigurations across your cloud resources
 
-Assess the configuration of your cloud resources, such as security groups, storage buckets, load balancers, and databases against configuration rules. Use the Datadog Agent to review local configuration information from servers, containers, and Kubernetes clusters against Datadog's OOTB Posture Management [Cloud][1] and [Infrastructure][2] detection rules.
+Strengthen your security posture and achieve continuous compliance by detecting, prioritizing, and remediating misconfigurations across all your cloud resources using Datadog's [out-of-the-box detection rules](#manage-out-of-the-box-and-custom-detection-rules). 
 
-View your cloud security posture at a high level with the [Posture Management][3] page. Examine the details of findings and analyze historical configurations with [Findings][4].
+View a high-level overview of your security posture on the [Summary page][3]. Examine the details of findings and analyze historical configurations with the [Security Findings Explorer][9].
 
-## Glossary
+{{< img src="security/cspm/summary_page.png" alt="Cloud Security Posture Management summary page" width="100%">}}
+
+## Maintain compliance with industry frameworks and benchmarks
+
+CSPM detection rules work with Datatog's out-of-the-box integration configurations. Each rule maps to one or more controls and requirements within a compliance standard or industry benchmark, such as the popular CIS compliance benchmarks for Docker and Kubernetes.
+
+[View compliance reports][10] for each framework that include details such as resources with the most failed findings, a comprehensive breakdown of the number of resources with pass/fail findings, and the top three high-severity rule failures. With this information, you can better understand whether your cloud infrastructure is in compliance with aspects of PCI, GDPR, HIPAA, and more.
+
+{{< img src="security/cspm/compliance_frameworks.png" alt="Cloud Security Posture Management compliance framewords" width="100%">}}
+## Manage out-of-the-box and custom detection rules
+
+Use [out-of-the-box detection rules][7] to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account. Customize the rules by defining [how each rule scans your environment][6], [create custom rules][8] (GCP only), and [set up real-time notifications for failed findings](#set-up-real-time-notifications).
+
+{{< img src="security/cspm/detection_rules.png" alt="Cloud Security Posture Management detection rules" width="100%">}}
+
+## Set up real-time notifications
 
+[Send real-time notifications][11] when a failed finding is generated for a detection rule, so that your teams can take action to mitigate the risk. Notifications can be sent to [Slack, email, PagerDuty, webhooks, and more][12].
```

Comment:
Up to you, but I'd rather be value oriented. "Failed finding is generated for a detection rule" read too technical.What about something like "Send real-time notifications when a new misconfiguration is detected in your environment..."

Reply:
Thanks for taking the time to review! I went ahead and incorporated your feedback into the PR.